### PR TITLE
Flush basket as we only support single item for now.

### PR DESCRIPTION
**Pull request by mjfrey**
Remove all existing lines in a basket as our basket page does not support managing of multiple items.

### CODE REVIEW

**Comment by mjfrey** [comment]
@vkaracic

**Comment by vkaracic** [comment]
Do we need a test for this? In which you add two different products to a basket and assert that only the second one survived.

**Comment by mjfrey** [comment]
Yes we do.  Let me add one.

**Comment by mjfrey** [comment]
@vkaracic add test.

File: ecommerce/extensions/basket/tests/test_utils.py [hunk 1]
```diff
@@ -56,16 +56,17 @@ def test_prepare_basket_without_voucher(self):
...
         self.assertFalse(basket.applied_offers())
 
-    def test_prepare_basket_with_multiple_quantity(self):
-        """ Verify a basket is returned and only contains a single quantity of product. """
-        product = ProductFactory()
-        basket = prepare_basket(self.request, product)
-        basket = prepare_basket(self.request, product)
+    def test_prepare_basket_with_multiple_products(self):
+        """ Verify a basket is returned and only contains a single product. """
+        product1 = ProductFactory(title='Product One', stockrecords__partner__short_code='test1')
```
Comment (vkaracic) [inline ecommerce/extensions/basket/tests/test_utils.py]: Since you're not using it anywhere in the test you can remove the title argument.

**Comment by vkaracic** [comment]
One nit, otherwise :+1:

**Pull request merged**